test/units/TEST-65-ANALYZE.sh

```diff
@@ -381,6 +381,29 @@ systemd-analyze verify /tmp/multi-exec-start.service
 echo 'ExecStart=command-should-not-exist' >>/tmp/multi-exec-start.service
 (! systemd-analyze verify /tmp/multi-exec-start.service)
 
+# Prevent regression from #20233 where systemd-analyze will return nonzero exit codes on warnings
+
+# Unit file with warning "Unknown key name 'foo' in section 'Unit', ignoring"
+cat <<EOF >/tmp/testwarnings.service
+[Unit]
+foo = bar
```

Comment:
We don't tend to put whitespace between unit assigments so please drop those. in other words make this:

```conf
[Unit]
Foo=Bar

[Service]
ExecStart=echo hello
```

Reply:
Fixed in latest version!